testsuite/lb_shear.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (C) 2010-2018 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import unittest as ut
+import numpy as np
+
+import espressomd.lb
+import espressomd.lbboundaries
+import espressomd.shapes
+
+"""
+Check the Lattice Boltzmann lid driven shear flow in a slab system
+by comparing to the analytical solution.
+
+
+"""
+
+
+AGRID = 0.5
+VISC = 1.4
+DENS = 2.3
+TIME_STEP = 0.01
+H = 30.
+SHEAR_RATE = 0.3
+
+LB_PARAMS = {'agrid': AGRID,
+             'dens': DENS,
+             'visc': VISC,
+             'fric': 1.0,
+             'tau': TIME_STEP}
+
+
+def shear_flow(x, t, nu, v, h, k_max):
+    """
+    Analytical solution for driven shear flow between two plates.
+
+    Parameters
+    ----------
+    x : :obj:`float`
+        Position from the left plane.
+    nu : :obj:`float`
+        Kinematic viscosity.
+    v : :obj:`float`
+        Shear rate.
+    h : :obj:`float`
+        Distance between the plates.
+    k_max: :obj:`int`
+        Maximum considered wave number.
```

Comment:
please add doc for returned value

Reply:
What do you mean by that?

Reply:
Sth. like
```
Returns
———-
:obj:`double` : Analytical velocity

```

Reply:
Ah ok

Reply:
i have 5. of https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard in mind